vagrant/Makefile

```diff
@@ -6,17 +6,35 @@ BUILDER_IMG = rpm-ostree-builder
 
 BUILDER_RUN = $(BUILDER_ARGS) $(BUILDER_IMG)
 
-buildimg:
+.PHONY: dockerwait buildimg build install ofsinstall check
+
+# necessary precaution in case we're operating on a freshly
+# booted VM
+dockerwait:
+	@echo "Waiting for docker service to start..."; \
+	timeout=10; \
+	while [ $$timeout -gt 0 ]; do \
+		if systemctl show -p SubState docker.service | grep -q =running; then \
```

Comment:
One pattern I've seen elsewhere is to do this as a higher order function, like:

`waitfor /bin/sh -c 'systemctl show -p SubState docker.service | grep -q =running'`

Like what one would do in ansible.